lib.php

```diff
@@ -152,38 +152,38 @@ function theme_boost_union_get_pre_scss($theme) {
 
     // Overwrite Boost core SCSS variables which need units and thus couldn't be added to $configurable above.
     // Set variables which are influenced by the coursecontentmaxwidth setting.
-    if (isset($theme->settings->coursecontentmaxwidth)) {
-        $scss .= '$course-content-maxwidth: '.$theme->settings->coursecontentmaxwidth.";\n";
+    if (get_config('theme_boost_union', 'coursecontentmaxwidth')) {
```

Comment:
Is there a reason that you removed the `isset()` and `!empty` calls in this and the subsequent lines and rely on implicit PHP magic?

I would prefer to keep the isset() and `!empty` calls here just just to be sure.

Reply:
Touché!
Thank you very much for your elaboration, @danowar2k !

Against the background of your explanations, I am fine with your proposed changes.